testutil/trackpr/pr/unticketed.go

```diff
@@ -0,0 +1,72 @@
+// Copyright © 2022 Obol Labs Inc.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the Free
+// Software Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of  MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+// more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program.  If not, see <http://www.gnu.org/licenses/>.
+
+// Package pr provides functions to check and process pull requests.
+//
+//nolint:wrapcheck,cyclop,forbidigo
+package pr
```

Comment:
i think we can add unticketed.go in testutil/trackpr package. no need for another sub package for one file